File: mario/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Enter the height: ");
    }
    while (height < 1 || height > 8);

    int space, hash;

    for (int i = 0; i < height; i++)
    {
        for (space = 0; space < height; space++)
        {
            if (i + space < height - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("  ");
        for (space = 0; space < height; space++)
        {
            if (height - i < space + 2)
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# TODO
if __name__ == "__main__":
    height = 0

    while True:
        height = get_int("Enter the height: ")

        for i in range(height):
            print(" " * (height - 1 - i), end="")

            print("#" * (1 + i), end="")

            print("  ", end="")

            print("#" * (1 + i), end="")

            print()

        if height >= 1 and height <= 8:
            break
